querybook/webapp/components/DataDocChartCell/DataDocChartComposer.tsx

```diff
@@ -91,6 +92,17 @@ const DataDocChartComposerComponent: React.FunctionComponent<
     const [displayStatementId, setDisplayStatementId] = React.useState(
         undefined
     );
+    const [chartReference, setChartReference] = React.useState<
+        ChartComponent<ChartComponentProps>
+    >();
+
+    // making sure legend color updated for bar/horiz bar/bubble charts
+    React.useEffect(() => {
+        if (chartReference) {
+            const chart = chartReference.chartInstance;
+            chart.update();
+        }
+    }, [values.coloredSeries, chartReference]);
```

Comment:
if you only want to update on values.coloredSeries change, then [values.coloredSeries] is good enough?

Reply:
omg i got confused with usememo/callback and thought it wouldnt change if i didnt add it 🤦